lagomjs-client/js/src/main/scala/com/lightbend/lagom/internal/client/WebSocketClient.scala

```diff
@@ -18,14 +23,21 @@ import org.scalajs.dom.WebSocket
 import org.scalajs.dom.raw.MessageEvent
 import play.api.http.Status
 
+import scala.collection.JavaConverters._
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 import scala.concurrent.Promise
 import scala.scalajs.js.typedarray.ArrayBuffer
 import scala.scalajs.js.typedarray.TypedArrayBuffer
 import scala.scalajs.js.typedarray.TypedArrayBufferOps._
+import scala.util.Failure
+import scala.util.Success
 
-private[lagom] abstract class WebSocketClient()(implicit ec: ExecutionContext) extends LagomServiceApiBridge {
+private[lagom] abstract class WebSocketClient(config: Config)(implicit ec: ExecutionContext, materializer: Materializer)
+    extends LagomServiceApiBridge {
+
+  // Defines the internal buffer size for the websocket when using ServiceCalls containing a Source as it's not possible to use backpressure in the JS websocket implementation
+  val maxBufferSize = Option(config.getInt("lagom.client.websocket.scalajs.maxBufferSize")).filter(_!=0).getOrElse(1024)
```

Comment:
I couldn't figure out how to use the `reference.conf` to define the default value :( the config stayed empty. Defining it directly in the ClientApplication by overriding `com.lightbend.lagom.scaladsl.api.LagomConfigComponent#config` works though